New Getting Started section (#4849)
DataDog/documentation: content/en/getting_started/_index.md

  text: "Introduction to Datadog"
---

> 

Reviewer: This page seems very empty now. It seems like we could take advantage of how many hits this page gets and still have a bit more overview content.

Author: Like the 3 pillars + 3 products workflow?

Reviewer 2: If we want something like this, we can probably add it later. Going to approve for now.

{{< whatsnext desc="This section includes the following topics:">}}

>     {{< nextlink href="/getting_started/agent" >}}<u>Agent</u>: Get Started over the Datadog Agent.{{< /nextlink >}}

Reviewer 2: ```suggestion
    {{< nextlink href="/getting_started/agent" >}}<u>Agent</u>: Get Started with the Datadog Agent.{{< /nextlink >}}
```

>     {{< nextlink href="/getting_started/application" >}}<u>Datadog Application</u>: Discover how to use Datadog UI: Dashboards, infrastructures, maps.. .{{< /nextlink >}}

Reviewer 2: ```suggestion
    {{< nextlink href="/getting_started/application" >}}<u>Datadog Application</u>: Discover how to use Datadog UI: Dashboards, infrastructure list, maps, etc.{{< /nextlink >}}
```

>     {{< nextlink href="/getting_started/integrations" >}}<u>Integrations</u>: Learn how to collect Metrics, Traces, and logs with Datadog Integrations.{{< /nextlink >}}

Reviewer 2: ```suggestion
    {{< nextlink href="/getting_started/integrations" >}}<u>Integrations</u>: Learn how to collect metrics, traces, and logs with Datadog integrations.{{< /nextlink >}}
```

>     {{< nextlink href="/getting_started/tracing" >}}<u>Tracing</u>: Ramp up on Tracing and distributed Tracing problematics.{{< /nextlink >}}

Reviewer 2: ```suggestion
    {{< nextlink href="/getting_started/tracing" >}}<u>Tracing</u>: Learn about tracing by setting up the Agent to trace a small application.{{< /nextlink >}}
```
